test/Unit/PowerShell/TestScripts/testBasicFunctionWithTriggerMetadata.ps1

```diff
@@ -7,5 +7,7 @@ param ($Req, $TriggerMetadata)
 
 # Used for logging tests
 Write-Verbose "a log"
+$cmdName = $MyInvocation.MyCommand.Name
```

Comment:
Do you have a test for PSScriptRoot as well?

Reply:
I forgot if I have a test. Will double check and add one if there isn't.

Reply:
Test case `InvokeFunctionWithSpecialVariableWorks` added.